core/services/ocr2/plugins/median/services.go

```diff
@@ -106,11 +107,20 @@ func NewMedianServices(ctx context.Context,
 		runResults,
 		chEnhancedTelem,
 	), ocrcommon.NewInMemoryDataSource(pipelineRunner, jb, pipeline.Spec{
-		ID:           jb.ID,
+		ID:           jb.ID, // why do we choose the same job id for getting juelsperfee number as the data source?
```

Comment:
Is this an issue we need to fix? or document in a ticket?

Reply:
This was just a question i had. I think I found the answer: they're part of the same job. ( will delete comment)